plugins/nanospell/plugin.js

```diff
@@ -937,8 +937,55 @@
 					}
 				}
 			);
+
+			range.shrink(CKEDITOR.SHRINK_TEXT);
```

Comment:
I wish they explained this better in the docs.

So bascially this means to shrink to something like

```
<p><strong>^asdf^</strong></p>
```
?